steppy/base.py

```diff
@@ -507,25 +596,19 @@ def _unpack(self, step_inputs):
                              for key, step_names in repeated_keys])
             raise StepsError(msg)
 
-    def _copy_transformer(self, step, name, dirpath):
-        self.transformer = self.transformer.transformer
-
-        original_filepath = os.path.join(step.exp_dir, 'transformers', step.name)
-        copy_filepath = os.path.join(dirpath, 'transformers', name)
-        logger.info('copying transformer from {} to {}'.format(original_filepath, copy_filepath))
-        shutil.copyfile(original_filepath, copy_filepath)
-
     def _prepare_experiment_directories(self):
         if not os.path.exists(os.path.join(self.exp_dir, 'outputs')):
             logger.info('initializing experiment directories under {}'.format(self.exp_dir))
             for dir_name in ['transformers', 'outputs']:
                 os.makedirs(os.path.join(self.exp_dir, dir_name), exist_ok=True)
+            for dir_name in ['train', 'inference']:
```

Comment:
@kamil-kaczmarek user defined output option is needed.

Reply:
Yes, I'll implement it tomorrow.